pilot/pkg/bootstrap/server.go

```diff
@@ -1224,7 +1226,49 @@ func (s *Server) initMeshHandlers() {
 	})
 }
 
-func (s *Server) initWorkloadTrustBundle() {
+func (s *Server) addIstioCAToTrustBundle(args *PilotArgs) {
+	// TODO: Will need to rework once CA code is refactored to update certs dynamically
```

Comment:
This PR doesn't have impact on the CA side. CA behavior keeps the same as before, i.e., CA only loads the root certs when Istiod is started. If we want CA to dynamically change the root based on the dynamics of the root certs secret, that's a separate work.

Reply:
Have removed this comment